modules/stampcollection.py

```diff
@@ -29,35 +33,31 @@ def reset_stamps(self):
         self.log.info(self.class_name, status="WIPING STAMP RECORDS")
 
         self.utils.clear_votes()
-        self.utils.update_vote(god_id, str(rob_id), self.red_stamp_value)
-
-    def update_vote(self, stamp_type, from_id, to_id, negative=False, recalculate=True):
+        self.update_utils()
+        self.calculate_stamps()
 
-        if to_id == stampy_id:
-            # votes for stampy do nothing
-            return
+    def update_vote(self, emoji: str, from_id: Union[int, str], to_id: Union[int, str],
+                    *, negative: bool = False, recalculate: bool = True):
```

Comment:
`*` that will eat all positional arguments is here on purpose?

Reply:
Yes. Passing booleans as positional arguments is not very legible (more so when there are many arguments and booleans are at the end). IMO it's better to make them keyword-only, so that it's clear what `True` and `False` mean whenever the function is called.

https://www.30secondsofcode.org/articles/s/javascript-boolean-trap

Reply:
right, badly phrased question on my part, I know why it might be a good idea to force keyword-only boolean parameters 🤔

I wanted to ask whether this code style change was already discussed with more people (in the previous PR or during office hours where I didn't make it yesterday) .. if yes, I am happy for this change, but if not, more people might have an opinion (e.g. if this is readable enough to @robertskmiles too, or whether it might deserve a code comment what is the meaning of `*` here)

Reply:
I don't think a comment is the way to go because there are likely more places, where it makes sense to make function arguments keyword-only.
If @robertskmiles approves and nobody else has a problem with that, I merge this PR.

Reply:
Yeah LGTM. If anyone wonders, it's the first thing in google
![image](https://user-images.githubusercontent.com/297544/191481743-3b665173-8d00-49cb-8af8-6c3d24252a88.png)